target/linux/ramips/dts/VOCORE2L.dts

```diff
@@ -0,0 +1,52 @@
+/dts-v1/;
+
+#include "VOCORE2.dtsi"
+
+/ {
+	model = "VoCore2-Lite";
```

Comment:
please add a compatible string here: `compatible = "vocore,vocore2lite", "vocore,vocore2", "mediatek,mt7628an-soc";`.

Would you please rename the file to VOCORE2LITE. It is often hard to get to which board a dts belongs due to the abbreviations.